support/cas-server-support-passwordless-ldap/src/test/java/org/apereo/cas/impl/account/LdapPasswordlessUserAccountStoreTests.java

```diff
@@ -55,5 +56,6 @@ public void verifyAction() throws Exception {
         assertTrue(user.isPresent());
         assertEquals("passwordlessuser@example.org", user.get().getEmail());
         assertEquals("123456789", user.get().getPhone());
+        assertEquals(false, user.get().getRequestPassword());
```

Comment:
This test is invalid; You should design a test case that check for this flag being `true`.